gemrb/core/GUI/GameControl.h

```diff
@@ -92,6 +92,7 @@ class GEM_EXPORT GameControl : public View {
 	Point screenMousePos;
 	Point vpOrigin;
 	bool updateVPTimer;
+	double formationBaseAngle;
```

Comment:
I think we of a mind to shift toward initializing member variables at their declaration (now that we use C++11)  so lets do that here by appending ` = 0.0`

```suggestion
	double formationBaseAngle = 0.0;
```

Reply:
Yes, indeed. It will enable us to remove most of the ugly constructor initializations for the classes with tons of fields. :)